### ISSUE TITLE
Upgraded to 0.9.3 and aws_ses_receipt_rule is always showing changes related to s3 action

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#702]
_This issue was originally opened by @levinse as hashicorp/terraform#13761. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

v0.9.3

```
~ aws_ses_receipt_rule.store
    s3_action.232311401.bucket_name:       "my-name-here" => ""
    s3_action.232311401.kms_key_arn:       "" => ""
    s3_action.232311401.object_key_prefix: "" => ""
    s3_action.232311401.position:          "1" => "0"
    s3_action.232311401.topic_arn:         "" => ""
    s3_action.348388136.bucket_name:       "" => "my-name-here"
    s3_action.348388136.kms_key_arn:       "" => ""
    s3_action.348388136.object_key_prefix: "" => ""
    s3_action.348388136.position:          "" => "0"
    s3_action.348388136.topic_arn:         "" => ""

```

### Expected Behavior
No changes shown since config wasn't changed

### Actual Behavior
Above changes are showing, even after applying them

**Comment by TheNotary** [comment]
x-post:  

I found out why terraform is behaving this way. Position 0 isn't accepted by aws, and I guess it re-writes it to position 1. Your terraform state file records the position property to be 1 instead of 0 which is fine. Then when you run plan again, terraform reads the "desired" position of "0" and see that this differs from the actual position and plans to make a modification that can't actually take place.

**Comment by rekcah78** [comment]
bug still present in last version, the position is not respected.
each terraform apply changes the position.

**Comment by bflad** [comment]
If the appropriate value is only 1 and greater, plan-time validation can be added to the attribute via: `ValidateFunc: validation.IntAtLeast(1),`

**Comment by bflad** [comment]
In version 1.27.0 of the AWS provider, releasing middle of this week, this argument will now appropriately validate that a value of 1 or greater was given at plan time.

**Comment by bflad** [comment]
This has been released in [version 1.27.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1270-july-11-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
